storage/innobase/include/dict0dict.h

```diff
@@ -1504,6 +1504,14 @@ class dict_sys_t
   bool load_sys_tables() noexcept;
   /** Create or check system tables on startup */
   dberr_t create_or_check_sys_tables() noexcept;
+
+  bool is_sys_table(table_id_t table_id) noexcept
```

Comment:
This is missing a `const` qualifier. Do we really need this definition in the `.h` file, or would an `inline` declaration suffice? The definition could be added to `storage/innobase/fsp/fsp0fsp.cc`, which contains the only invocation of this function.